d2l-icon-button.html

```diff
@@ -1,28 +1,81 @@
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../d2l-offscreen/d2l-offscreen-shared-styles.html">
 <link rel="import" href="d2l-icon.html">
 <link rel="import" href="d2l-icon-button-shared-styles.html">
 <link rel="import" href="d2l-icon-button-behavior.html">
 
 <dom-module id="d2l-icon-button">
 	<template strip-whitespace>
-		<style include="d2l-icon-button-shared-styles d2l-offscreen-shared-styles">
-			:host > span {
-				@apply(--d2l-offscreen);
+		<style include="d2l-icon-button-shared-styles">
+			:host {
+				display: inline-block;
 			}
 		</style>
-		<d2l-icon icon="[[icon]]"></d2l-icon>
-		<span><content></content></span>
+		<button class="d2l-icon-button"
+			disabled={{disabled}}
+			title={{title}}
+			autofocus={{autofocus}}
+			form={{form}}
+			formaction={{formaction}}
+			formenctype={{formenctype}}
+			formmethod={{formmethod}}
+			formnovalidate={{formnovalidate}}
+			formtarget={{formtarget}}
+			name={{name}}
+			type={{type}}
+			value={{value}}
```

Comment:
I think you can get away with just 1-way binding (i.e. `[[disabled]]`) since nothing is going to be interacting directly with the `<button>` element anymore to modify it. That'll be a performance gain.